string object_id
geometry_msgs/PoseStamped target_pose
